# Gets the Blobs structure

---
cmvision/Blobs  blobs

